plumbing/proofs/proofs.go

```diff
@@ -0,0 +1,35 @@
+package proofs
+
+import (
+	"github.com/filecoin-project/go-filecoin/proofs"
+	"github.com/filecoin-project/go-filecoin/proofs/sectorbuilder"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Provides the sector builder for proofs.
+type sectorBuilderer interface {
+	SectorBuilder() sectorbuilder.SectorBuilder
+}
+
+// Proofs is plumbing for proof calculation and verification.
+type Proofs struct {
```

Comment:
I do want to avoid exposing the whole sector builder interface. I was tempted to push the request/response marshalling down (essentially removing the structs from the sector builder interface) in which case this would be a vanishingly thin wrapper. I don't really follow why exposing the parts separately in plumbing is undesirable, but I'll try pushing it up to porcelain.

Reply:
I have exposed SectorBuilder on plumbing and moved `CalculatePoSt` wrapper to porcelain.